Earthfile

```diff
@@ -238,7 +238,7 @@ earthly:
     ARG VERSION="dev-$EARTHLY_TARGET_TAG_DOCKER"
     ARG EARTHLY_GIT_HASH
     ARG DEFAULT_BUILDKITD_IMAGE=docker.io/earthly/buildkitd:$VERSION # The image needs to be fully qualified for alternative frontend support.
-    ARG BUILD_TAGS=dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork dfheredoc
+    ARG BUILD_TAGS=dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork dfheredoc forceposix
```

Comment:
The https://github.com/jessevdk/go-flags package automatically uses the `/` for options when detect is building on Windows. This is, of course, undesired in the case of Earthfile. The `forceposix` makes go-flags always use the dashes for the options.